ReactAndroid/src/main/java/com/facebook/react/modules/accessibilityinfo/AccessibilityInfoModule.java

```diff
@@ -108,9 +108,10 @@ public String getName() {
 
   @TargetApi(Build.VERSION_CODES.LOLLIPOP)
   private boolean getIsReduceMotionEnabledValue() {
-    float defaultAnimationScale = Float.parseFloat(Settings.Global.TRANSITION_ANIMATION_SCALE);
-    float animationScale = Settings.Global.getFloat(mContentResolver, defaultAnimationScale);
-    return animationScale == 0f;
+    String value =
+        Settings.Global.getString(mContentResolver, Settings.Global.TRANSITION_ANIMATION_SCALE);
+
+    return value != null && value.equals("0.0");
   }
```

Comment:
These changes are unrelated to the PR but they're necessary to get RN to compile locally